OpenRA.Mods.Cnc/Traits/Spreads.cs

```diff
@@ -0,0 +1,74 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Cnc.Traits
+{
+	[Desc("A trait that controls forest fires. Any actor with this trait enabled has a chance to affect others with this trait.")]
+	public class SpreadsInfo : ConditionalTraitInfo
+	{
+		[Desc("The chance an actor is affected by an adjacent actor that is already affected.")]
```

Comment:
```suggestion
		[Desc("The chance this actor is affected by an adjacent actor that is already affected.")]
```

Reply:
Nope, the trait isn't passive. The spreader controls the probability.